src/utils/run-status.js

```diff
@@ -0,0 +1,44 @@
+import { localStorageLastRunEndTime } from '../config';
+import { normalizeTimestamp } from '../components/workflow/workflow-utils/format';
+
+/**
+ * Handle the latest run status by comparing end times
+ * @param {string} endTime End time of the latest run
+ * @returns {boolean} True if the latest run is newer than the last stored run
+ */
+export const handleLatestRunStatus = (endTime) => {
```

Comment:
I found that the name doesn't align with the Boolean value it returns. When I read `handleLatestRunStatus`, I expect it to return some kind of value or data.

Reply:
yeah you are right. it could be something like `isLatestRun` or `isNewRun`